docs/reference/commandline/run.md

```diff
@@ -65,6 +65,7 @@ parent = "smn_cli"
       -p, --publish=[]              Publish a container's port(s) to the host
       --pid=""                      PID namespace to use
       --privileged=false            Give extended privileges to this container
+      --read-bps-device=""       Specifies upper limit on READ rate from the device
```

Comment:
Oh, asked too soon; I see the example below. We probably need a "format" in the description, e.g.

```Markdown
The format is `<device-path>:<number>[unit]`. `number` must be greater than `0`. Unit is optional and can be `b` (bytes per second), `mb` (megabyte per second)....
```

Thinking out loud here, because I don't know if "unit" is optional, and which units are allowed :relaxed: